ts/src/program/namespace/account.ts

```diff
@@ -202,9 +215,14 @@ export class AccountClient<T = any> {
           {
             memcmp: {
               offset: 0,
```

Comment:
Can we add an additional offset param to use here?

Reply:
fo sho